script/install-nginx.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2018-2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
```

Comment:
```suggestion
set -euo pipefail
```
Nitpick, this can be done with a oneliner.